specs/integer_palindrome_check_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require_relative '../lib/integer_palindrome_check'
+
+describe "integer palindrome check" do
+  describe "basic tests" do
+    describe "test 1" do
+      it "palindrome input with even digit count" do
+        is_palindrome(1001).must_equal true
+      end
+    end
+
+    describe "test 2" do
+      it "palindrome input with odd digit count" do
+        is_palindrome(1234321).must_equal true
+      end
+    end
+
+    describe "test 3" do
+      it "not a palindrome" do
+        is_palindrome(1728371).must_equal false
+      end
+    end
+  end
+
+  # if the parameter is an object, check for nil
+  describe "nil object input test" do
+    it "nil object is not an integer" do
```

Comment:
I could see this test going into the edge cases section.

Reply:
done!